topics/dev/tutorials/core-contributing/tutorial.md

```diff
@@ -28,19 +30,27 @@ requirements:
         - architecture
 ---
 
-# Contributing a New Feature to Galaxy Core
+# Introduction
+{:.no_toc}
```

Comment:
```suggestion
```

generally we avoid the first section title since it's a bit superfluous, anyone will interpret the bit above the agenda as "introduction" or abstract, and everything after agenda as the real core of the tutorial.

Reply:
The previous title was the same as the title of the tutorial, which looked redundant. I was following the example of other tutorials (I'm clicking through other tutorials at https://training.galaxyproject.org/training-material/topics/dev/ - it appears Introduction is quite common as the first section). Besides, unlike, for examle, the Debugging tutorial, here we have several paragraphs introducing the content in more detail, so it seems like a proper section, just without a hands-on component. I don't mind dropping the title if you think that's more appropriate, but, again, that would be a question for @jmchilton.

Reply:
We do the same across many admin tutorials, e.g. [ansible-galaxy](https://training.galaxyproject.org/training-material/topics/admin/tutorials/ansible-galaxy/tutorial.html), just no section at all, not title or introduction. the introduction h1 in e.g. the [debugging](https://training.galaxyproject.org/training-material/topics/dev/tutorials/debugging/tutorial.html#introduction) feels a bit useless.

The redundant title is definitely sub-optimal though, +1 on removing that.
 **If you'd like to keep `introduction` for now, fine, no objection.**

For context: I'm working on better tutorial previews, and for the vast majority of tutorials people write 2-3 paragraphs of introduction which behaves a *lot* more like a tutorial's abstract, something we can easily show in link previews or elsewhere in the interface, so I've been slowly removing a lot of those headers.